doc/proposals/kubernetes-1.17.md

```diff
@@ -10,7 +10,7 @@ creation-date: 2019-12-19
 last-updated: 2019-12-19
 status: implementable
 see-also:
-  - https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.17.md#changes
```

Comment:
We probably should put this fix in a separate PR (which will have to merge first) in case we have to revert the /etc/passwd change for some reason.

Reply:
OKs.